Public/Src/Utilities/Utilities/Tracing/EventId.cs

```diff
@@ -312,7 +312,8 @@ public enum EventId
         PipRetryDueToExitedWithAzureWatsonExitCode = 317,
         // Reserved = 318,
 
-        // Free slot 319,
+        PipIpcFailedDueToInfrastructureError = 319,

```

Comment:
>PipIpcFailedDueToInfrastructureError [](start = 8, length = 36)

Please add this to the Tracing\EventIds.cs in the Scheduler project, not here.

Reply:
done